mmv1/third_party/tgc_next/pkg/tfplan2cai/converters/services/compute/compute_instance.go

```diff
@@ -1523,31 +1521,17 @@ func expandComputeInstance(project string, d tpgresource.TerraformResourceData,
 
 	for i := 0; i < attachedDisksCount; i++ {
 		diskConfig := d.Get(fmt.Sprintf("attached_disk.%d", i)).(map[string]interface{})
-		disk, err := expandAttachedDisk(diskConfig, d, config)
+		disk, err := expandAttachedDiskTgc(diskConfig, d, config)
```

Comment:
Looks the new function re-used a lot of code from the deleted `expandAttachedDisk`. Could we keep the old function and merge these functions together?

Reply:
Sure, modified the original function.